src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -454,10 +471,8 @@ Start, Stop, Restart
 
          Run the workflow either by:
 
-         * Pressing the play button in the Cylc GUI. Then, ensuring that
-           "Cold Start" is selected within the dialogue window, pressing the
-           "Start" button.
-         * Running the command ``cylc play runtime-tutorial``.
+         * TUI/GUI: pressing play for the latest run.
```

Comment:
```suggestion
         * GUI: pressing play for the latest run.
         * Tui: pressing enter on the workflow and selecting "play".
```

Reply:
The user will have already played the first run in a previous step, so I tried to keep this short. Happy to commit though if you want

Reply:
That's fine, no need to repeat the same advice, maybe we could just link back to it?